.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: true
+
+language: generic
+
+git:
+  depth: 5
+  
+cache:
+  directories:
+  - "$HOME/.stack"
+  - "$HOME/build/NatalyaSlepkova/haskell-course-assistant/.stack-work"
```

Comment:
User name should be `ChShersh` instead of `NatalyaSlepkova` because it will eventually run under my Travis account.